modules/nf-core/universc/launch/main.nf

```diff
@@ -0,0 +1,62 @@
+process UNIVERSC_LAUNCH {
+    tag "$meta.id"
+    label 'process_medium'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using the Cell Ranger tool. Please use docker or singularity containers."
+        conda (params.enable_conda ? "hcc::cellranger=3.0.2" : null)
+    }
+    container "tomkellygenetics/universc:1.2.4"
```

Comment:
Hm, we have the tendency to mirror these in nfcore/<whatever> in cases where the containers are not available via Bioconda. Might be something to discuss in the main thread too.